iaso/api/org_unit_types/serializers.py

```diff
@@ -101,15 +111,35 @@ def get_sub_unit_types(self, obj: OrgUnitType):
             context=self.context,
         ).data
 
+    def get_allow_creating_sub_unit_types(self, obj: OrgUnitType):
+        unit_types = obj.allow_creating_sub_unit_types.all()
+        app_id = self.context["request"].query_params.get("app_id")
+        if app_id is not None:
+            unit_types = unit_types.filter(projects__app_id=app_id)
```

Comment:
this is to return only the sub type for the app_id so the mobile one only have the correct one, do I understand correctly ? Can we add a comment?